src/datastore_mad/remotes/lxd_downloader.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+get_tag_name () {
+    version=`oned -v | grep "is distributed" | awk '{print $2}'`
```

Comment:
Not sure if it's better to depend on a file that could even be deleted instead of the stable output of a command that will always succeed. @rsmontero, what do you think? Although...if we use that file to get the version number at the frontend for other tasks it could be better to unify.